lib/exchange_api_web/controllers/order_controller.ex

```diff
@@ -2,9 +2,61 @@ defmodule ExchangeApiWeb.OrderController do
   use ExchangeApiWeb, :controller
 
   def index(conn, _params) do
-    json(conn, %{status: "** shrug das orders **"})
+    orders = Exchange.open_orders(get_ticker())
+    json(conn, %{ status: elem(orders, 0), data: elem(orders, 1) })
```

Comment:
the use of `elem/2` makes this harder to understand. I would suggest using pattern matching such as:
```
{status, data} = Exchange.open_orders(get_ticker())
json(conn, %{status: status, data: data})
```